share/man/man4/ure.4

```diff
@@ -53,16 +53,8 @@ if_ure_load="YES"
 .Sh DESCRIPTION
 The
 .Nm
-driver provides support for USB Ethernet adapters based on the RealTek
-RTL8152, RTL8153/RTL8153B, and RTL8156/RTL8156B USB Ethernet controllers,
-as well as USB Ethernet PHYs provided by
-.Xr rgephy 4 .
-.Pp
-NICs based on the RTL8152 are capable of 10 and 100Mbps.
-NICs based on the RTL8153 or provided by
-.Xr rgephy 4
-are capable of 10, 100, and 1000Mbps.
-NICs based on the RTL8156 are capable of 10, 100, 1000, and 2500Mbps operation.
+driver provides support for various RealTek-based USB Ethernet adapters listed
+in the HARDWARE section below.
```

Comment:
```suggestion
driver supports the Realtek USB Ethernet Controller family.
```
Digging deeper and looking at how Realtek (which I just realized is not spelled with a capital T) describes them [^1], maybe we should specify the actual family with even more specificity and terseness?

The FreeBSD manual pages spec says that we list hardware in the hardware section, so we probably shouldn't mention that to improve semantic density.

[^1]: https://www.realtek.com/Download/List?cate_id=585